index.bs

```diff
@@ -3207,12 +3207,13 @@ Note: The {{AuthenticatorTransport}} enumeration is deliberately not referenced,
     for instance, <code>-7</code> for "ES256" and <code>-257</code> for "RS256".
 
     The COSE algorithms registry leaves degrees of freedom to be specified by other parameters in a [=COSE key=]. In order to promote interoperability, this specification makes the following additional guarantees of [=credential public keys=]:
-        1. Keys with algorithm ES256 (-7) MUST specify P-256 (1) as the [=crv=] parameter.
-        1. Keys with algorithm ES384 (-35) MUST specify P-384 (2) as the [=crv=] parameter.
-        1. Keys with algorithm ES512 (-36) MUST specify P-521 (3) as the [=crv=] parameter.
-        1. Keys with algorithm EdDSA (-8) MUST specify Ed25519 (6) as the [=crv=] parameter.
+        1. Keys with algorithm ES256 (-7) MUST specify P-256 (1) as the [=crv=] parameter and MUST NOT use the compressed point form.
+        1. Keys with algorithm ES384 (-35) MUST specify P-384 (2) as the [=crv=] parameter and MUST NOT use the compressed point form.
+        1. Keys with algorithm ES512 (-36) MUST specify P-521 (3) as the [=crv=] parameter and MUST NOT use the compressed point form.
+        1. Keys with algorithm EdDSA (-8) MUST specify Ed25519 (6) as the [=crv=] parameter. (These always use a compressed form in COSE.)
```

Comment:
I suggest changing "compressed form" to "compressed point form", for consistency sake.